core/src/main/java/org/opensearch/sql/analysis/model/CatalogName.java

```diff
@@ -14,7 +14,7 @@
 @Getter
 public class CatalogName {
 
-  public static final String DEFAULT_CATALOG_NAME = ".opensearch";
+  public static final String DEFAULT_CATALOG_NAME = "@opensearch";
```

Comment:
Not sure if I miss the context but why default catalog `opensearch` has to prefixed with some special character ?

Reply:
Since it is auto-created and not uploaded to catalog.json. I am using a special character to differentiate.

Reply:
Okay. I doubt should we expose this to customer. Because in future this default catalog may be some config in user session. This special naming will be broken.

Reply:
removing then.